### PR TITLE
Integration of ratings post

### PR DESCRIPTION
**Pull request by Asam237**
https://user-images.githubusercontent.com/34966088/154643120-19c01221-88fc-41c1-94c8-e6f4a228b1b9.mp4

### CODE REVIEW

File: frontend/components/MyInput.vue [hunk 1]
```diff
@@ -36,5 +40,21 @@ import Vue from 'vue'
...
   props: ['title', 'myStyle', 'myInput'],
+  data() {
+    return {
+      nameInput: '',
+      nameTextArea: '',
+    }
+  },
+  methods: {
+    onChangeInput() {
+      console.log('Ma console input', this.nameInput)
```
Comment (elhmn) [inline frontend/components/MyInput.vue:51]: left over ?

File: frontend/components/MyInput.vue [hunk 2]
```diff
@@ -36,5 +40,21 @@ import Vue from 'vue'
...
+      this.$store.dispatch('selectValueSalary', this.nameInput)
+    },
+    onChangeTextArea() {
+      console.log('Ma console textarea', this.nameTextArea)
```
Comment (elhmn) [inline frontend/components/MyInput.vue:55]: left over ?

**Review by elhmn — APPROVED** [review]
Great work, I just left few comments about the `console.log`

**Comment by Theryx** [comment]
Love it🙌🏼 
This is the most exciting project I've worked on

**Review by Zaker237 — APPROVED** [review]

**Pull request merged**